unime/src/i18n/nl-NL/index.ts

```diff
@@ -233,10 +233,10 @@ const nl = {
   },
   BADGE: {
     DETAILS: {
-      VALID: '',
-      ISSUED_BY: '',
-      DESCRIPTION: '',
-      METADATA: '',
+      VALID: 'Geldig',
+      ISSUED_BY: 'Uitgegeven door',
+      DESCRIPTION: 'Beschrijving',
+      METADATA: 'Metagegevens',
```

Comment:
I'd go for `Metadata` here. Its still a valid Dutch word and I think it is more commonly used than `Metagegevens`.